apps/codeforafrica/src/lib/api.netlify-cms/index.js

```diff
@@ -5,6 +5,8 @@ import getOurPartners from "./getOurPartners";
 import getPartners from "./getPartners";
 import getCmsProjects from "./getProjects";
 import getTeam from "./getTeam";
+import getFooter from "./sections/getFooter";
+import getHeader from "./sections/getHeader";
```

Comment:
Why not just expose a single `getSettings`? I don't see the value of separate `getFooter`/`getHeader`.

----

`getSettings()` could return an object or [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) `{ footer: {...}, header: { ... } }` if calling `getSettings("footer")` etc., is too much.